src/test/ui/parser/lex-bad-char-literals-4.stderr

```diff
@@ -1,8 +1,8 @@
-error: character literal may only contain one codepoint: '●
+error: unterminated character literal: '●  
   --> $DIR/lex-bad-char-literals-4.rs:4:5
    |
 LL |     '●
-   |     ^^
+   |     ^^^^
```

Comment:
Shouldn't this stop after one codepoint?

Reply:
I guess either consuming one code point, or consuming until newline work the same. I've switched the strategy because it's easier in the new impl. 

Also, "unclosed literal" error makes more sense in this particular case :P